pkgs/applications/networking/nextcloud-client/default.nix

```diff
@@ -26,15 +26,15 @@
 
 mkDerivation rec {
   pname = "nextcloud-client";
-  version = "3.6.4";
+  version = "3.6.6";
 
   outputs = [ "out" "dev" ];
 
   src = fetchFromGitHub {
     owner = "nextcloud";
     repo = "desktop";
     rev = "v${version}";
-    sha256 = "sha256-ZtDgm9xlBQflVXsxjt4bFmRby6ni0wjaGYaoiEWH9Q0=";
+    sha256 = "sha256-P3LSgrcMZZM0OY3yQz8t3Cf5spJJTB+JTIpoT9U3+xc=";
```

Comment:
```suggestion
    hash = "sha256-P3LSgrcMZZM0OY3yQz8t3Cf5spJJTB+JTIpoT9U3+xc=";
```

Reply:
I still prefer sha256 personally, I think some tools like nixpkgs-update don't support updating `hash` yet.

Reply:
nixpkgs-update should be good, see #211957.